packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,161 @@
+| shouldAddStartEndMargin  | `boolean`                              | `false`                                  | If true, adds extra margin before and after the first/last slides so that they align in center, this prop can be used to achive carousel which is in [GST page](https://razorpay.com/gst-number-search/pan/) |          |
+| showIndicators           | `boolean`                              | `true`                                   | Toggles the visibility of indicators                                                                                                                                                                         |          |
+| showOverlay              | `boolean`                              | `false`                                  | Toggles the visibility of overlay                                                                                                                                                                            |          |
+| overlayColor             | `BrandColorTokens, SurfaceColorTokens` | `surface.background.level1.highContrast` | Changes the color of the overlay, so that consumers can blend the overlay with the background color                                                                                                          |          |
+| navigationButtonPosition | `bottom,side`                          | `bottom`                                 | Sets the position of navigation button                                                                                                                                                                       |          |
+| navigationButtonSpacing  | `SpacingTokens`                        | `spacing.4`                              | Spacing between navigation button and slides when bleed is set to none                                                                                                                                       |          |
+| navigationButtonStyle    | `filled,stroke`                        | `filled`                                 | Styles to be used depending on the emphasis you want to give to the navigation buttons                                                                                                                       |          |
+| indicatorVariant         | `gray,white,blue`                      | `gray`                                   | Variants to be used depending on the background and the emphasis you want to give to the indicator.                                                                                                          |          |
+| onChange                 | `(slideIndex: number) => void`         | `undefined`                              | Callback which gets fired everytime a slide changes, useful for analytic events (only runs on user interaction not on autoPlay)                                                                              |          |
```

Comment:
how about `ref`? for example, I want to scroll into a particular carousel item?

Reply:
> @anuraghazra I mentioned POS just as an example, there are cases in the past in which the active item of the carousel was controlled through an external element.

Yeah usecase hain, we can probably expose `goToSlide` method via a ref.

Reply:
yes we shall